frontend/src/views/Signup/components/UserInfoSSOStep/UserInfoSSOStep.tsx

```diff
@@ -69,12 +54,13 @@ export const UserInfoSSOStep = ({
   const [organizationName, setOrganizationName] = useState("");
   const [organizationNameError, setOrganizationNameError] = useState(false);
   const [attributionSource, setAttributionSource] = useState("");
-  const [errors, setErrors] = useState<Errors>({});
   const [isLoading, setIsLoading] = useState(false);
   const { t } = useTranslation();
   const { mutateAsync: selectOrganization } = useSelectOrganization();
 
   useEffect(() => {
+    const randomPassword = crypto.randomBytes(32).toString("hex");
+    setPassword(randomPassword);
```

Comment:
Would like a bit more context on this part

Reply:
For signup of SSO user, this password doesn't matter. Its a random string. Soon in server we will be doing the same property.